src/components/shared-settings/settings.ts

```diff
@@ -18,3 +21,28 @@ export const disableOpenNewTabInBackgroundSettings = createNewSettings<boolean>(
             "Many of Maskbook features relies on this behavior. Disable this behavior will limit Maskbook's functionality",
     },
 )
+
+const createProxiedSettings = (key: string) => {
+    const target: { [key: string]: ValueRef<string> | string } = {}
```

Comment:
Should this `target` defined in the global scope?

Reply:
no. it is only for current settings.